packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -346,6 +346,11 @@ class CombinedHintsEditor extends React.Component<CombinedHintsEditorProps> {
     };
 
     handleHintRemove: (i: number) => void = (i: number) => {
+        // eslint-disable-next-line no-alert
```

Comment:
I imagine that this linting rule was made for user-facing code and it's okay here because it's internal users?

Reply:
Correct. Ideally this would be a Wonder Blocks modal confirm thingy, but given this is only for internal users I think it's ok (we do this in webapp in a few places). IMHO, this is alot better than nothing for now.